File: 06-DataBinding-VB/Student.vb
''' <summary>
''' A simple business object standing in for a row you would normally pull from a database or
''' business layer. History holds HTML markup - that field is what gets bound to the editor.
''' </summary>
<Serializable>
Public Class Student
    Public Property Name As String = ""
    Public Property RollNumber As Integer
    Public Property History As String = ""
End Class

File: 06-DataBinding-CS/Student.cs
namespace DataBinding;

/// <summary>
/// A simple business object standing in for a row you would normally pull from a database or
/// business layer. History holds HTML markup - that field is what gets bound to the editor.
/// </summary>
[Serializable]
public class Student
{
    public string Name { get; set; } = "";
    public int RollNumber { get; set; }
    public string History { get; set; } = "";
}
